### DISCUSSION TITLE
be start failed

### DISCUSSION THREAD

**Discussion by tianqin69** [apache/doris#8183] [A - General / Q&A]
bin/start_be.sh: line 102:  7300 Illegal instruction     $LIMIT ${DORIS_HOME}/lib/palo_be "$@" >> $LOG_DIR/be.out 2>&1 < /dev/null

**Comment by dataalive** [comment]
cat /proc/cpuinfo

check whether cpu supports avx2 instrustions. if not ,pls use build again with no avx2 set.

like 

USE_AVX2=0 sh build.sh
